loops-and-arrays.js

```diff
@@ -22,12 +25,21 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+for (let i = 0; i < numbers.length ; i++) {
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = numbers[i]*2;
```

Comment:
Formatting: let op de spaties rondom de operators (`*`). Je kunt in dit geval de vermenigvuldiging ook als volgt opschrijven: `numbers[i] *= 2`. Beiden zijn goed.